src/pages/EventAdministration/components/EventFileSender.tsx

```diff
@@ -0,0 +1,90 @@
+import { useState } from 'react';
+import { useSendFilesToAttendees } from 'hooks/Event';
+import { useSnackbar } from 'hooks/Snackbar';
+import { FileUploader } from 'react-drag-drop-files';
+
+// Material-UI
+import { Button, IconButton, List, ListItem, ListItemIcon, ListItemText, Box, Typography } from '@mui/material';
+import { AttachFileRounded, CloseRounded } from '@mui/icons-material/';
+
+// Project components
+import Dialog from 'components/layout/Dialog';
+
+export type EventFileSenderProps = {
+  eventId: number;
+};
+
+const EventFileSender = ({ eventId }: EventFileSenderProps) => {
+  const [dialogOpen, setDialogOpen] = useState(false);
+  const showSnackbar = useSnackbar();
+  const [files, setFiles] = useState<File[]>([]);
+  const fileTypes = ['JPG', 'PNG', 'GIF', 'PDF', 'DOCX'];
```

Comment:
`FILE_TYPES` er en konstant og kan stå utenfor komponenten. Tenker og at vi kun skal sende ut pdf og evt docx
```suggestion
const FILE_TYPES = ['JPG', 'PNG', 'GIF', 'PDF', 'DOCX'];

const EventGiftCardSender = ({ eventId }: EventGiftCardSenderProps) => {
  const [dialogOpen, setDialogOpen] = useState(false);
  const showSnackbar = useSnackbar();
  const [files, setFiles] = useState<File[]>([]);
```